creepy/subprocess/pypen.py

```diff
@@ -80,12 +82,27 @@ def __init__(self, args, hash: Optional[str] = None, serializable: bool = False,
         try:
             self._send, self._recv = secure_alice(send, recv, make_cipher=self._save_and_make_cipher)
         except Exception:
-            self.detach()
+            self._detach()
+        self._bind_interface()
 
     @property
-    def pid(self):
+    def _pid(self):
         return self._process.pid
 
+    def _bind_interface(self):
+        interface = self._request('_interface')
+        for func_name, func in interface.items():
+            params = []
+            for name, kind, has_default in func['params']:
+                param = Parameter(name, kind, default=_default_value() if has_default else Parameter.empty)
```

Comment:
As I don't see any other checks against `_default_value` why not `default=None`?

Reply:
`default=None` reads like there is no default value. On the other hand I don't know why. If you don't like it, I will use `default=None` then.